drivers/filesystems/fastfat_new/fatprocs.h

```diff
@@ -40,6 +40,13 @@ Module Name:
 
 
 #ifdef __REACTOS__
+// Downgrade unsupported NT6.2+ features.
+#define MdlMappingNoExecute 0
+#define NonPagedPoolNx NonPagedPool
+#define NonPagedPoolNxCacheAligned NonPagedPoolCacheAligned
+#define POOL_NX_ALLOCATION 0
```

Comment:
Can you `#undef POOL_NX_ALLOCATION` (and `MdlMappingNoExecute`) please, so that it doesn't break once our DDK headers get these defines?

Reply:
Added `#if`.

Reply:
Should be `#undef`, not `#if`

Reply:
I saw the request, but I don't know/understand why.
What is wrong with `#if NTDDI_VERSION < NTDDI_WIN8`?
What would be wrong with `#ifndef ...`?

Reply:
ReactOS will add these defines and enum members to its DDK headers sooner or later, to be compatible with the real WDK. The point was to change this code such that it wouldn't break at that point. Your change doesn't achieve that.

Reply:
> sooner or later

Now, before this PR, may be the right time?

NB:
I don't have WDK 8+, so can't look at that myself.

> this driver would use the real values and compile but stop working

Ah, you plan to update the PSDK in a way ROS Core will not support: I couldn't guess that.

Reply:
> match the latest public version

I understand and agree with that, though it may be a bit tricky, as NT6.2+ MS headers do not support NT5.2 anymore, do they?
Anyway, no point debating this here: new headers were not added, PR is merged now.

> build things written against MS headers

It seems this PR/driver is such a case, and well...

> compile & behave correctly when built against MS's headers

Does ROS build system have an option to do that? It would help working on that goal.